can-define.js

```diff
@@ -64,7 +64,7 @@ module.exports = define = ns.define = function(objPrototype, defines, baseDefine
 	// for any value that has a default value.
 	replaceWith(objPrototype, "_data", function() {
 		var map = this;
-		var data = {};
+		var data = Object.create(null);
```

Comment:
I disagree with the view of the person reporting that issue; in my estimation, accidental enumerable properties on the default prototype chain are more likely and more frustrating than not having default Object functions on reserved properties.  This instance above was meant to be boyscouting; I can remove it here, but if you took it to a consensus we'd likely opt for using this pattern for plain key/value map objects going forward.